policyengine_canada/variables/gov/cra/tax/income/credits/climate_action/climate_action_incentive_basic.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_canada.model_api import *
+
+
+class climate_action_incentive_basic(Variable):
+    value_type = float
+    entity = Household
+    label = "Canada Climate Action amount per child under 19"
```

Comment:
Labels shouldn't include policy parameters - the max age should be a policy parameter
```suggestion
    label = "Canada Climate Action amount per child"
```